Tempura/Core/ViewController+Containment.swift

```diff
@@ -16,10 +16,43 @@ extension ViewController {
     view.addSubview(child.rootView)
     child.didMove(toParent: self)
   }
-  
+
+  /// Replace a ViewController (and its rootView) to the last child of self.
+  /// You must provide a ContainerView inside self.rootView, the latest view of that container view
+  /// is replaced with the rootView of the child ViewController with a cross-dissolve transition
+  public func transition<View: ViewControllerModellableView>(
+    to child: ViewController<View>,
+    in view: ContainerView,
+    duration: Double = 0.3,
+    options: UIView.AnimationOptions = [],
```

Comment:
As default I would use `.transitionCrossDissolve` but remember to use them below.